content/en/docs/started/workstation/minikf-aws.md

```diff
@@ -0,0 +1,191 @@
++++
+title = "Deploy Kubeflow using MiniKF on AWS"
+description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via AWS"
+weight = 40
++++
+
+![MiniKF latest
+version](https://www.arrikto.com/wp-content/uploads/2020/11/minikf-latest-version-aws.svg
+"MiniKF latest version")
+
+This guide describes how to launch a MiniKF instance on AWS. MiniKF is a single
+VM solution on the AWS Marketplace, and installs:
+
+- Kubernetes (using Minikube)
+- Kubeflow
+- Kale, an orchestration and workflow tool for Kubeflow that enables you to run
+  complete data science workflows starting from a notebook
+  ([GitHub](https://github.com/kubeflow-kale))
+- Arrikto's Rok, a data management software for data versioning and
+  reproducibility
+
+### Installing MiniKF on AWS
+
+To install MiniKF on AWS, follow the steps below:
+
+1. Go to the [MiniKF page](https://aws.amazon.com/marketplace/pp/B08MBGH311) on
+   the AWS Marketplace
+1. Click on the **Continue to Subscribe** button
+
+   <img src="/docs/images/minikf-aws/minikf-aws-page.png"
+       alt="Continue to Subscribe"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click on the **Accept Terms** button
+
+   <img src="/docs/images/minikf-aws/minikf-aws-accept-terms.png"
+       alt="Accept Terms"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Wait for a few minutes and the click on **Continue to Configuration**
+
+   <img src="/docs/images/minikf-aws/minikf-aws-continue-configuration.png"
+       alt="Continue to Configuration"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Select a **Region**. We recommend you keep the default values for **Delivery
+   Method** and  **Software Version**. Then click **Continue to Launch**
+
+   <img src="/docs/images/minikf-aws/minikf-aws-configuration.png"
+       alt="Configure MiniKF"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Here you can review your configuration details. Don’t forget to view the
+   **Usage Instructions** and keep them handy, as you are going to need them
+   later.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-configuration-details.png"
+       alt="Configuration Details"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. You can also select how to launch MiniKF or change the EC2 instance type. We
+   recommend you leave the default value for **Choose Action**. We recommend
+   that you keep the default **EC2 Instance Type** (m5.2xlarge) or choose an
+   even more powerful instance. Choosing a VM with reduced specs may make it
+   impossible to train ML models.
```

Comment:
```suggestion
1. Go to the [MiniKF page](https://aws.amazon.com/marketplace/pp/B08MBGH311) on
   the AWS Marketplace.
1. Click on the **Continue to Subscribe** button:

   <img src="/docs/images/minikf-aws/minikf-aws-page.png"
       alt="Continue to Subscribe"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Click on the **Accept Terms** button:

   <img src="/docs/images/minikf-aws/minikf-aws-accept-terms.png"
       alt="Accept Terms"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Wait for a few minutes and the click on **Continue to Configuration**:

   <img src="/docs/images/minikf-aws/minikf-aws-continue-configuration.png"
       alt="Continue to Configuration"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Select a **Region**. It's recommended to keep the default values for **Delivery
   Method** and  **Software Version**. When you're finished, click **Continue to Launch**.

   <img src="/docs/images/minikf-aws/minikf-aws-configuration.png"
       alt="Configure MiniKF"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Review your configuration details. Don’t forget to view the
   **Usage Instructions** and keep them handy, as you are going to need them
   later.

   <img src="/docs/images/minikf-aws/minikf-aws-configuration-details.png"
       alt="Configuration Details"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. You can also select how to launch MiniKF or change the EC2 instance type. You're
   recommended to leave the default value for **Choose Action** and
   keep the default **EC2 Instance Type** (m5.2xlarge) (or choose an
   even more powerful instance). Choosing a VM with reduced specs may make it
   impossible to train ML models.
```

Reply:
Active voice is the way to go. I wouldn't change it to passive. Please take a look at this post by @sarahmaddox : https://ffeathers.wordpress.com/2016/07/30/eliminating-the-zombie-vulnerability-removing-passive-voice-from-the-docs/